library/L2_HAL/sensors/battery/test/ltc4150_test.cpp

```diff
@@ -1,38 +1,60 @@
 #include "L2_HAL/sensors/battery/ltc4150.hpp"
 #include "L4_Testing/testing_frameworks.hpp"
+#include "L1_Peripheral/interrupt.hpp"
+#include "L1_Peripheral/gpio.hpp"
+#include "utility/units.hpp"
 
 namespace sjsu
 {
 #define APPROX_EQUALITY(expected, actual, resolution) \
   CHECK(-resolution < (actual - expected));           \
   CHECK((actual - expected) < resolution);
 
+auto GetLambda(sjsu::InterruptCallback & isr)
+{
+  return [&isr](sjsu::InterruptCallback callback,
+                [[maybe_unused]] sjsu::Gpio::Edge edge) { isr = callback; };
```

Comment:
```suggestion
                sjsu::Gpio::Edge) { isr = callback; };
```
If you simply do not give the argument a name, it will automatically be ignored, which means you can remove the `[[maybe_unused]]` attribute.